src/package.ps1

```diff
@@ -303,6 +315,7 @@ function RemovePackage {
 	if ($null -ne $err) {
 		throw $err
 	}
+	Remove-Item ($Pkg | ResolvePackageRefPath) -ErrorAction Ignore
```

Comment:
I wasn't sure if here or at the end of the `UninstallPackage` was a better spot for this. Thoughts?